src/types/user.ts

```diff
@@ -81,16 +79,15 @@ export type User = t.TypeOf<typeof User>;
 export const SpidUser = t.intersection([
   t.interface({
     authnContextClassRef: SpidLevel,
+    dateOfBirth: t.string,
```

Comment:
Are we sure IDPs provide such value? Are we sure it's always valued?

Reply:
Good catch, thanks. Yes it could be missing during the metadata propagation to all the IDPs.
Fixed in 8d6ca2a